source/index.js

```diff
@@ -32,7 +32,8 @@ function __processStatOutput(filePath, result) {
         path: path.resolve(path.dirname(filePath), name),
         isFile: () => result.isFile(),
         isDirectory: () => result.isDirectory(),
-        size: result.size || 0
+        size: result.size || 0,
+        mtime: result.mtime || 0
```

Comment:
I think the `mtime` value should be a timestamp or null perhaps: `result.mtime || null`. If it's a timestamp a value of `0` would be quite far out :)